Viewer/src/viewer/defaultViewer.ts

```diff
@@ -85,7 +85,7 @@ export class DefaultViewer extends AbstractViewer {
 
         let pointerDown = <PointerEvent>event.event;
         if (pointerDown.button !== 0) return;
-        var element = (<HTMLElement>event.event.srcElement);
+        var element = (<HTMLElement>event.event.target);
 

```

Comment:
Was it tested on edge and chrome?

Reply:
Yes

Reply:
You rock

Reply:
Should check with @RaananW if this is an issue.